### DISCUSSION TITLE
Is it possible to have multiple cursors without any matching text?

### DISCUSSION THREAD

**Discussion by azzamsa** [smoka7/multicursors.nvim#98] [Q&A]
Hi there, 👋

Sometimes I want to place multiple cursors in a file without matching any specific text. I use this feature a lot with [magnars/multiple-cursors.el](https://github.com/magnars/multiple-cursors.el).

For example, if I have the following lines:

```
foo
bar
baz
```

How can I add a cursor to the beginning of each of these lines, like this:

```
|foo
|bar
|baz
```

Is this possible?

Thank you!

**Comment by ghost** [comment]
This is already a built in vim feature, using rectangular selection.
